content/concepts/contributing.md

```diff
@@ -33,7 +33,7 @@ Before you start coding right away, please follow those basic guidelines:
 A typical code contribution in any Ocean Protocol repository would go as follows:
 
 1. As an external developer, fork the respective repo and push to your own fork. Ocean core developers push directly on the repo under `oceanprotocol` org.
-2. We follow [Trunk Based Development](https://trunkbaseddevelopment.com) so work in feature branches, branched off from the `main` branch. For naming use `feature/your-feature` or `feature/23` for new features and `fix/your-fix` or `fix/23` for bug fixes, referring to the issue number.
+2. You should create a new branch for your changes. The naming convention for branches is: `issue001/short-feature-description`. The issue number `issue001` needs to reference the GitHub issue that you are trying to fix. The short feature description helps to quickly distinguish your branch among the othe branches in play.
```

Comment:
it's `issue001-short-feature-description` with a dash 😄

Reply:
and there's a typo in "othe" branches

Reply:
I've corrected the spelling mistake. I used a slash instead of a dash because I thought it made it a bit clearer which issue the branch was about, but I'm happy to change that if you disagree

Reply:
Everyone agreed on dash, let's go with that. (It's my personal preference too).

Reply:
ok sure